### ISSUE TITLE
Decepticon Quake and Titan Masters

### ISSUE THREAD

**Issue by hen** [tftcg/questions#3]
If a Titan Master is KO'd while Quake is in body mode, will the KO'd character's Head take a damage as well, or does Quake's ability happen before the Head is deployed?

**Comment by TechnomagusPrime** [comment]
Unless WotC changes the rules with Wave 5, deploying a Head when a Body is KOed is a triggered ability.  Therefore, it falls under active player/nonactive player order.  If the active player controls Quake and KOs a defending body, the Head is deployed after Quake's damage.  If the defending player controls both Quake and the KOed Body, they can choose to have Quake deal damage before or after the Head deploys.  If the attacker controls the KOed Body and the defender controls Quake (KOed via Backfire or PTT, for example), the Head will deploy before Quake deals damage.

**Comment by hen** [comment]
Agreed. Though it feels very complex for newcomers so I'm intrigued as to whether WotC will make an update to the Expanded Rules for Head deploys.
